microshift_storage/understanding-persistent-storage-microshift.adoc

```diff
@@ -6,8 +6,15 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
+Managing storage is a distinct problem from managing compute resources. {product-title} uses the Kubernetes persistent volume (PV) framework to allow cluster administrators to provision persistent storage for a cluster. Developers can use persistent volume claims (PVCs) to request PV resources without having specific knowledge of the underlying storage infrastructure.
+
 include::modules/storage-persistent-storage-overview.adoc[leveloffset=+1]
 
+[id="additional-resources_understanding-persistent-storage-microshift"]
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/storage/understanding-persistent-storage#pv-access-modes_understanding-persistent-storage[Access modes for persistent storage]
```

Comment:
@kcarmichael08 Yes, it is a link because we cannot xref across topic maps. The content does apply to 4.12 OCP because MicroShift versioning moves in parallel. Hmmm, I don't think I can evergreen point to something on the Portal because of the pagination there (but I'll double check).